README.md

```diff
@@ -22,23 +20,11 @@ This project is emphatically beginner-friendly so, if you're interested in poten
 
 This site runs on **Ruby 2.0.0-p353** and **Rails 4.0.0** (upgraded from Ruby 1.9.3 and Rails 3.2.12 in February 2014)
 
-#### Nitrous.io Setup
-
-We suggest that Windows users who can't migrate to Linux and newbies who are having trouble setting up their environments just use [nitrous.io](https://www.nitrous.io/join/GRrt3VYaHE8?utm_source=nitrous.io&utm_medium=copypaste&utm_campaign=referral) or a similar cloud-based environment like [Cloud 9](https://c9.io/). 
-
-For Nitrous:
-1. [Follow along with this gist](https://gist.github.com/afshinator/8035821) to do the setup on Nitrous.
-2. Don't forget to populate your test database with `$ rake db:test:prepare`
+#### Installing the Site on your Local Machine
```

Comment:
and one final instruction under those 2,
3. Use the appropriate operating system focused guide below to install the site